OnixFirebaseChat/OnixFirebaseChat/Codebase/Services/ONXFirebaseService.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ONXFirebaseService.swift
+//  OnixFirebaseChat
+//
+//  Created by Anton Dolzhenko on 02.12.16.
+//  Copyright © 2016 Onix-systems. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+import Firebase
+import FirebaseAuthUI
+import FirebaseFacebookAuthUI
+import FirebaseGoogleAuthUI
+import FirebaseTwitterAuthUI
+import TwitterKit
+
+final class ONXFirebaseService: NSObject {
```

Comment:
Class name came from: https://habrahabr.ru/post/273455/, regarding to: https://github.com/marinbenc/ReactiveWeatherExample/blob/master/WhatsTheWeatherIn/WeatherAPIService.swift